lms/envs/common.py

```diff
@@ -2701,3 +2701,10 @@
     'options': {},
 }
 PROCTORING_SETTINGS = {}
+
+#### Custom Courses for EDX (CCX) configuration
+
+# This is an arbitrary hard limit.
+# The reason we introcuced this number is because we do not want the CCX
+# to compete with the MOOC.
+CCX_MAX_STUDENTS_ALLOWED = 200
```

Comment:
FYI, I am not sure what kind of string/constants edX support in site configuration files. I got below feedback when i put maximum number of students per page in common.py. They suggest me to put constant in app.
 
> this is also probably not the right place for this setting to live. We may also want to just pick a number and not allow it to be site configurable, because this configuration file is 2682 (!!!!) lines long, which is insane.